docs/general/serializers.md

```diff
@@ -74,6 +74,30 @@ def blog
 end
 ```
 
+### Polymorphic Relationships
+
+Polymorphic relationships are serialized just by specifying the relationship. For example:
```

Comment:
Only possible issue is with serializable type in a has many... Eg has_many -> [user, post] what is root in json and tyoe in json api with explicit and implicit association serializer